notebooks/sentiment_analysis.ipynb

```diff
@@ -0,0 +1,714 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "multilingual_pipeline([\n",
+    "    \"How are you today?\",  # How are you today?\n",
+    "    \"I'm a little tired, I didn't sleep well, but I hope it gets better\"\n",
+    "    # \"I'm a little tired, I didn't sleep well, but I hope it gets better\n",
+    "])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "While it is a bit optimistic about our first prompt, it's guess is given with a fairly low confidence, and it identifies the second prompt as neutral with a median score of 3.\n",
+    "\n",
+    "Now let's translate our prompts and ask it the same questions in French:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ambiguous_in_french = [\n",
+    "    \"Comment vas-tu aujourd'hui?\",\n",
+    "    \"Je suis un peu fatigue, je n'ai pas bien dormi mais j'espere que la journee s'ameliore\",\n",
+    "]\n",
+    "multilingual_pipeline(ambiguous_in_french)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The model gives very similar results in both languages!\n",
+    "\n",
+    "Now let us revisit our first model, could it also work with this multi-lingual input?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(sentiment_pipeline(ambiguous_prompts))\n",
+    "print(sentiment_pipeline(ambiguous_in_french))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Unsurprisingly it cannot, its prediction are not stable across the two languages. However this model did not perform very well on the ambiguous prompts. If we re-use some of the more precise English-only models and run them on more obvious prompts: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "simple_french_input = [\n",
+    "    \"Ce film est excellent\",  # This film is excellent\n",
+    "    \"Le produit ne marche pas du tout\"  # The tool does not work at all\n",
+    "]\n",
+    "print(tweet21_pipeline(simple_french_input))\n",
+    "print(multilingual_pipeline(simple_french_input))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this case we see the multi-lingual model correctly predict strongly positive and negative labels, while the other model predicts a positive message (correct) and a neutral (expected LABEL_0)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Other models\n",
+    "\n",
+    "Models can be finetuned to extract different classes from text. The `bhadresh-savani/distilbert-base-uncased-emotion` checkpoint is a DistilBERT checkpoint tuned to identify the emotion associated with a prompt:"
```

Comment:
The wording "The `bhadresh-savani/distilbert-base-uncased-emotion` checkpoint is a DistilBERT checkpoint" is a bit clunky. Can we change one "checkpoint" to "model"? Or remove one?